### ISSUE TITLE
"Selection is source" check box for Patch tool

### ISSUE THREAD

**Issue by hgs54371** [photopea/photopea#604]
When patching something, selecting source first is more intuitive, just like Healing brush tool

**Comment by hgs54371** [comment]
But the name "source" "target" are reversed

**Comment by photopea** [comment]
I don't think so. E.g. for "source", you select the area that you want to patch. Then you drag to choose the source, from wich it should be patched.

**Comment by hgs54371** [comment]
You choose the source material to patch the target area, like affinity "selection is source" I understand this way better 
But in photoshop the term is used reversely, which always confused me, right now it is termed like in photoshop
